### ISSUE TITLE
0.34.0 update has broken all highlighting.

### ISSUE THREAD

**Issue by albell** [abe33/atom-pigments#222]
I'm not getting any highlighting on hex or rgba() values:

![screen shot 2016-08-29 at 10 02 52 am](https://cloud.githubusercontent.com/assets/1142025/18059860/df398a1e-6dcf-11e6-843b-c364718e0352.png)

I've tried uninstalling and reinstalling. Nothing. This was working fine before the update.

**Comment by tsmuse** [comment]
I'm seeing the same thing in Atom 1.9.9. No highlighting for any color code or SASS variable like I had in previous versions. I'm also now getting inconsistent and incorrect highlighting for named colors in SASS. 
<img width="294" alt="screenshot 2016-08-30 11 57 45" src="https://cloud.githubusercontent.com/assets/45876/18102871/0fe190a6-6ea9-11e6-9d46-b19904e5f600.png">

**Comment by abe33** [comment]
Hi there, I can't really tell what happened in your case, the tests are still all green (I wouldn't have published a new version otherwise) and I'm not experiencing that on the various project I work on.

Here's what I get with @tsmuse example: 
![_default_scss_ __users_c_nehemie_development_gsoi_newho](https://cloud.githubusercontent.com/assets/247140/18124361/3d1500ac-6f72-11e6-95b1-7603cb1303b0.png)

**Comment by albell** [comment]
Highlghting still wasn't working this morning. Then I upgraded to 1.10.0, and now highlighting is working again. Great, problem solved I guess!

**Comment by tsmuse** [comment]
Same here. Upgraded to 1.10.0 and everything is back to normal. ¯\_(ツ)_/¯

**Comment by abe33** [comment]
No pb, I'm not really sure what could have triggered that issue on 1.9 that is no longer failing on 1.10. If I can find some time I'll try to investigate and see if it's an easy fix or not.

**Comment by letops** [comment]
I am having the exact same problem here with my SCSS and CSS files (the only places where I use colors). I have the latest Atom version (1.10.2) and the latest Pigments version (0.37.0). 
I have reinstalled Pigments and cleared the configuration but still the same. 

:(

![screenshot from 2016-10-05 12 32 48](https://cloud.githubusercontent.com/assets/9222108/19124216/e1a66eee-8af7-11e6-8a0e-05c7c91e7815.png)

**Comment by abe33** [comment]
@letops Could you try running the `Pigments: Reload` command to see if it gets any better?

**Comment by mbacon40** [comment]
I am having this same issue as well. `Pigments: Reload` does not help.

**Comment by letops** [comment]
Sorry, @abe33 I wasn't available due to work.
I actually tried reloading the package, reinstalling it and deleting my previous configuration file (cson).

Only the "find colors" feature continues working as expected, but the "convert" or "copy as" features have stopped working as well.
The "pigment" feature only works when I reopen a file, if I modify a line it doesn't get updated.
![screenshot from 2016-10-17 15 02 50](https://cloud.githubusercontent.com/assets/9222108/19453419/cf80f59a-947a-11e6-9152-77cdefd055ca.png)
 
By now I can just make the wild guess that pigments is collisioning with another of my installed packages, so just if you want to know, I'll list all my installed packages:

- atom-beautify@0.29.13
- atom-django@0.3.2
- atom-elixir@0.2.1
- atom-jinja2@0.5.0
- atom-material-syntax@0.4.6
- atom-material-ui@1.3.6
- autocomplete-python@1.8.6
- browser-plus@0.0.76
- file-icons@1.7.22
- hyperclick@0.0.39
- language-babel@2.44.1
- language-elixir@0.16.0
- linter@1.11.18
- linter-csslint@1.3.3
- linter-elixirc@1.1.11
- linter-erlc@0.3.0
- linter-htmlhint@1.3.0
- linter-js-yaml@1.2.5
- linter-jscs@4.1.0
- linter-markdown@3.0.2
- linter-pep8@1.3.2
- linter-scss-lint@3.0.4
- linter-xmllint@1.3.6
- merge-conflicts@1.4.4
- minimap@4.25.0
- minimap-linter@1.6.0
- minimap-pigments@0.2.1
- minimap-split-diff@0.3.2
- pigments@0.37.0
- script@3.10.1
- split-diff@1.0.5
- svg-preview@0.10.0
- terminal-plus@0.14.5

Once again, sorry for the trouble.

**Comment by abe33** [comment]
@letops no it's me that is sorry that it doesn't work for you. Could you post a snippet of the styles that are not highlighted properly so that I can try a few things on my config?

**Comment by letops** [comment]
@abe33 I am gald to tell you that the problem was resolved, although I still dont know why I can assure you that one of this packages is the culprit behind this. I updated this Atom packages today and your package has cero problems and works as sweet and smooth as before. (I don't believe it was a Core package problem as this issue would have got more participants)
 
- language-babel 2.45.0
- linter-htmlhint 1.3.1
- minimap 4.25.6
- script 3.11.1
- split-diff 1.0.6

I am sorry for worrying you and once again thank you for an excelent package.
Cheers mate!
